man/fi_efa.7.md

```diff
@@ -53,8 +53,9 @@ The following features are supported:
   registrations on the DGRAM endpoint.
 
 *Memory registration modes*
-: The RDM endpoint does not require memory registration and the
-  *FI_EP_DGRAM* endpoint only supports *FI_MR_LOCAL*.
+: The RDM endpoint does not require memory registration for send and receive
```

Comment:
Maybe add a note that: "by default RDM end point does not require memory registration, but if application specify FI_MR_LOCAL in MR mode, RDM end point will adjust its behavior and use the descriptor provided by application".

Reply:
Sure, I've added that.